packages/fast-components/src/slider-label/slider-label.styles.ts

```diff
@@ -0,0 +1,58 @@
+import { css } from "@microsoft/fast-element";
+import { SystemColors } from "../styles/system-colors";
+
+export const SliderLabelStyles = css`
+    :host {
```

Comment:
this should use the `display` function from `../styles/display` so it can benefit from the "hidden" styles and any other universal styles we want to add in the future.

Reply:
I think I did this correctly, might want to double check it. I imported display and set it as I thought it should be for slider-label